ocpi-toolkit-2.2.1/src/main/kotlin/com/izivia/ocpi/toolkit/modules/cdr/CdrsEmspServer.kt

```diff
@@ -35,7 +35,7 @@ class CdrsEmspServer(
 
         transportServer.handle(
             method = HttpMethod.POST,
-            path = basePathSegments
+            path = basePathSegments + listOf(VariablePathSegment("cdrId"))
         ) { req ->
             req.httpResponse {
                 service
```

Comment:
`.postCdr(...)` should have `cdrId` in its params (even if it is already included in Cdr object)

Reply:
```suggestion
                 path = basePathSegments
```

was correct